sss/sss_table.py

```diff
@@ -54,59 +143,120 @@ class SSS(Base):
     annual_self_sufficiency_wage = Column(
         'annual_self_sufficiency_wage', Float)
     emergency_savings = Column('emergency_savings', Float)
-    
     miscellaneous_is_secondary = Column(
         'miscellaneous_is_secondary', Boolean)
     health_care_is_secondary = Column(
         'health_care_is_secondary', Boolean)
-
     analysis_is_secondary = Column(
         'analysis_is_secondary', Boolean)
+
+class ARPA(Base):
+    """
+    state : str column 
```

Comment:
This needs a one line description and then a `Attributes` heading and a line of dashes before you start listing the attributes. And as for the primary table, the column attributes should be listed as SQL Alchemy column types. and we don't need empty lines between the attributes.